parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -166,8 +168,12 @@ type InputDescription = Text
 
 makeLenses ''LoopState
 
+-- replacing the old read/write scalar Lens with "peek" Getter for the NonEmpty
+currentPath :: Getter (LoopState m v) Path.Absolute
+currentPath = currentPathStack . to Nel.head
```

Comment:
In place of the old read/write `currentPath` lens, I added a getter that peeks at the head of `currentPathStack`, so the usage sites don't have to change.